trailblazer/store/crud/update.py

```diff
@@ -315,3 +317,16 @@ def update_user_token(self, refresh_token: str, user_id: int) -> None:
         user.refresh_token = refresh_token
         session: Session = get_session()
         session.commit()
+
+    def update_analysis_status(self, analysis_id: int, status: TrailblazerStatus) -> None:
+        analysis: Analysis | None = self.get_analysis_with_id(analysis_id)
+        analysis.status = status
+        session: Session = get_session()
+        session.commit()
```

Comment:
This looks a bit strange, but I guess it's how you do it if you don't pass it as a parameter 🤔

Reply:
I agree, potentially we could just directly import the `Session` and use it as described here https://github.com/Clinical-Genomics/trailblazer/pull/423/commits/cf838778fcad580369e55230caf429304031bdf8.

I'll leave it for now, but might be something we want to revisit.